src/thx/Arrays.hx

```diff
@@ -1160,6 +1161,17 @@ Finds the min element of the array given the specified ordering.
 
     return Options.toFailure(Nel.fromArray(collisions), m);
   }
+
+  public static function toStringMap<V>(arr: ReadonlyArray<Tuple<String, V>>): StringMap<V> {
```

Comment:
It does still compile if I change the return type to Map<String, V>. Where I'm using it in thx.schema I had been forced to use the more specific StringMap type, because it complained about it not conforming but I'll see if I can change thx.schema and get it to work.